### PR TITLE
Realtime priority for realtime cyclers

### PR DESCRIPTION
**Pull request by oleflb**
## Introduced Changes

Makes cyclers with realtime kind also use a realtime scheduler with priority 20. This should (?) fix issues of other cyclers stealing ressources from Control.

Fixes #

## ToDo / Known Issues
- We need the capability `sys_cap_nice` on the `hulk` binary before merging this PR. Otherwise we can only launch with sudo

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test
* Check if process starts up and robot behaves normally

### CODE REVIEW

File: crates/code_generation/src/cyclers.rs [hunk 1]
```diff
@@ -379,7 +379,27 @@ fn generate_consumer_identifiers(cyclers: &Cyclers) -> Vec<Ident> {
         .collect()
 }
 
-fn generate_start_method() -> TokenStream {
+fn generate_start_method(cycler_kind: CyclerKind) -> TokenStream {
+    let scheduler_tokens = match cycler_kind {
+        CyclerKind::Perception => quote! {
+            unsafe {
+                let priority = libc::sched_param {
+                    sched_priority: 10,
```
Comment (h3ndrk) [inline crates/code_generation/src/cyclers.rs:387]: This is 10 but the PR description mentions 20?

File: crates/framework/Cargo.toml
Comment (h3ndrk) [inline crates/framework/Cargo.toml:1]: Sort alphabetically and add final newline according to POSIX standard?

**Review by h3ndrk — CHANGES_REQUESTED** [review]

File: crates/framework/Cargo.toml
Comment (oleflb) [reply crates/framework/Cargo.toml:1]: done

File: crates/code_generation/src/cyclers.rs (hunk 1 above)
Comment (oleflb) [reply crates/code_generation/src/cyclers.rs:387]: hoppla, meant to write 10 in the description, that way it has lower priority compared to hula

**Review by h3ndrk — COMMENTED** [review]
Code schaut gut zu mir :handshake: We're testing tomorrow

**Review by h3ndrk — APPROVED** [review]

**Pull request merged**